docs/uploading.md

```diff
@@ -15,17 +15,13 @@ Bind a [file input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/in
 {{/let}}
 ```
 
-After a file is added to your queue, the `onFileAdded` callback will be called. You may then upload the file to a URL by calling `file.upload()`. This returns a promise that is resolved when the file has finished uploading, or is rejected if the file couldn't be uploaded.
+After a file is added to your queue, the `onFileAdded` callback will be called. You may then upload the file to a URL by calling `file.upload()`. This returns a promise that is resolved when the file has finished uploading.
```

Comment:
Is the promise not rejecting anymore at all? Is it resolving even if upload fails? I'm confused by removing that part from the docs.

Reply:
I got a bit carried away here – you can disregard this.